Clean up FAQs sections and add new Get Started FAQs (#219)
OffchainLabs/arbitrum-docs: arbitrum-docs/partials/_troubleshooting-bridging-partial.md

### How do I move assets between One and Nova? {#how-do-i-move-assets-between-one-and-nova}

> <p>Both Arbitrum One and Arbitrum Nova run as layers on top of Ethereum. Thus, you can always move assets between the two chains in two steps by going "through" Ethereum. In other words: withdraw your assets on Arbitrum One to Ethereum and then deposit them onto Nova, or conversely, withdraw your assets from Nova on to Ethreum and then deposit them on to Arbitrum One. These steps can all be done at <a href="https://bridge.arbitrum.io/">https://bridge.arbitrum.io/</a>.</p>

Reviewer: ```suggestion
<p>Both Arbitrum One and Arbitrum Nova run as layers on top of Ethereum. Thus, you can always move assets between the two chains in two steps by going "through" Ethereum. In other words: withdraw your assets from Arbitrum One to Ethereum and then deposit them onto Nova, or conversely, withdraw your assets from Nova on to Ethreum and then deposit them on to Arbitrum One. These steps can all be done at <a href="https://bridge.arbitrum.io/">https://bridge.arbitrum.io/</a>.</p>
```

> <p>Some <a href="https://portal.arbitrum.one/#bridges">third party bridges</a> support transferring between Arbitrum One and Nova directly; (remember that third party bridges are created by third parties; DYOR!)</p>

Reviewer: correct me if I'm wrong but as far as I know, there is no fast bridge that supports direct transfers between one and nova

Author: [Multichain](https://app.multichain.org/) supports transfers between One and Nova, for ETH and a few more tokens, but I'm afraid is the only one that supports that atm.
Do you think we should remove that paragraph? ETH transfers are interesting though

<p></p>

### What fees do I have to pay when bridging funds from L1 to L2? {#what-fees-do-i-have-to-pay-when-bridging-funds-from-l1-to-l2}
<p>When bridging over tokens from L1 to L2, you will have to sign one or two transactions with their corresponding fees:</p>

> <ol><li>If your bridging a token for the first time, you'll sign one <strong>approval transaction</strong>.</li>

Reviewer: ```suggestion
<ol><li>If you are bridging a token for the first time, you'll sign one <strong>approval transaction</strong>.</li>
```

<li>In all cases, you'll sign a <strong>deposit transaction</strong> that will send your tokens to the Bridge.</li>
</ol>
<p></p>

> <p>Keep in mind that the approval transaction needs to be executed once per token and wallet. This means that if you bridge again the same token from the same wallet, you won't have to pay that transaction. However, if you bridge the same token from a different wallet, you will have to pay that transaction again.</p>

Reviewer: ```suggestion
<p>Keep in mind that the approval transaction needs to be executed once per token and wallet. This means that if you bridge the same token from the same wallet again, you won't have to pay for that transaction. However, if you bridge the same token from a different wallet, you will have to pay for that transaction again.</p>
```

> <p>Keep in mind that the approval transaction needs to be executed once per token and wallet. This means that if you bridge the same token from the same wallet again, you won't have to pay for that transaction. However, if you bridge the same token from a different wallet, you will have to pay for that transaction again.</p>

Reviewer 2: maybe " _at least_  once per token and wallet" and "you _probably_  won't have to pay for that transaction" since a transactions amount could exceed the initial approval (I don't know offhand what amount the UI currently approves)

<p></p>

<p>In any case, the bridge and your wallet will guide you through the process, showing the transaction(s) that you need to sign in order to have your tokens bridged to Arbitrum.</p>

<p></p>

> ### When I initiate withdrawal from Arbitrum, how long does it take before I receive my funds? {#when-i-initiate-withdrawal-from-arbitrum-how-long-does-it-take-before-i-receive-my-funds}

Reviewer: ```suggestion
### How long does it take before I receive my funds when I initiate a withdrawal from Arbitrum chains (One and Nova)? {#when-i-initiate-withdrawal-from-arbitrum-how-long-does-it-take-before-i-receive-my-funds}
```

> <p><a href="https://developer.arbitrum.io/faqs/what-if-dispute#okay-okay-but-if-were-just-talking-about-an-l2-to-l1-message-and-assuming-theres-no-disputes-how-long-between-the-time-the-message-is-initiated-and-when-i-can-execute-it-on-l1-is-it-exactly-one-week"></a></p>

Reviewer 2: Looking at this again, it might make sense to open with the caveat that many users opt to use 3rd party fast bridges so don't experience this delay (or something)
